app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/MapViewModel.java

```diff
@@ -140,6 +146,11 @@ public void mapReady() {
         mDisplayMapEntitiesInteractor = mDisplayMapEntitiesInteractorFactory.create(this);
```

Comment:
I know it's legacy but I think this also should be done by an inner class implementing the DisplayMapEntitiesInteractor.Displayer interface instead of having the MapViewModel implement it. As you did with the new interactor.